tests/Unit/ClientTest.php

```diff
@@ -45,7 +45,9 @@ public function it_should_create_the_vehicles_scope_with_its_own_path(): void
         $this->baseUri->method('getPath')
             ->willReturn('/aaa/bbb/ccc');
 
-        $this->baseUri->method('withPath')
+        $this->baseUri
+            ->expects($this->once())
```

Comment:
Small fix-up: fixes a warning of no assertions being performed as there was previously no expectation that this was called, and therefore it didn't add to the assertion counter.